doc/contributors-guide/making-and-submitting-changes.md

```diff
@@ -10,6 +10,10 @@ Everyone is welcome to help improve the content of the Git and Github training r
 * Send a pull request pointing to the develop branch of the original learning resource repo.
 * Wait for review of your changes from the repo admin(s).
 
+{% hint style="info" %}
```

Comment:
Very well done @ManasviGoyal 
And the PR title too.
Changes approved!

Reply:
Thank you.